src/css/custom.css

```
@@ -274,7 +282,7 @@ h1 {
 }
 .user-survey {
     width: 100%;
-    height: 6175px;
+    min-height: 600px;
```

I changed this one explicitely on request to have the whole survey in view and not in a viewport:

```suggestion
    min-height: 600px;
    height: 6175px;
```